tests/tests_files_files.yml

```diff
@@ -113,11 +120,19 @@
       failed_when: not __result.stat.exists
 
     - name: Check the filter 0
-      command: /bin/grep 'input(type="imfile" file="\/var\/log\/inputdirectory0\/\*\.log" tag="files_input0")' '{{ __test_inputfiles_conf0 }}'
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space